open-source.md

```diff
@@ -22,7 +22,7 @@ A software philosophy that promotes free redistribution and access to source cod
 
 ## Intermediate Open Source
 
-* Read [8 steps to fixing other people's code](http://rdd.me/tsf7khhg)
+* Read [8 steps to fixing other people's code](https://thoughtbot.com/blog/8-new-steps-for-fixing-other-peoples-code)
```

Comment:
Oops, the original URL was supposed to be http://drnicwilliams.com/2007/06/01/8-steps-for-fixing-other-peoples-code/, but it's also dead. Oh no.

Reply:
From the web archive: https://web.archive.org/web/20180719201947/http://drnicwilliams.com/2007/06/01/8-steps-for-fixing-other-peoples-code/

Yeah, the mentioned tools are not relevant anymore. Our blog post is the spiritual successor.